src/reporters/udp_sender.js

```diff
@@ -132,10 +132,6 @@ export default class UDPSender {
         this._client.send(thriftBuffer, 0, thriftBuffer.length, PORT, HOST);
         this._reset();
 
-        if (testCallback) {
```

Comment:
This is correct - this method is effectively synchronous, UNLESS there is a way to pass a callback to `this._client.send`. If there is not, then there should be no callbacks in this method. So remove it from the arguments too.